pkg/scheduler/apis/config/testing/defaults/defaults.go

```diff
@@ -287,3 +287,141 @@ var PluginConfigsV1beta2 = []config.PluginConfig{
 		},
 	},
 }
+
+// PluginsV1beta3 default set of v1beta3 plugins.
+var PluginsV1beta3 = &config.Plugins{
+	QueueSort: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.PrioritySort},
+		},
+	},
+	PreFilter: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.NodeResourcesFit},
+			{Name: names.NodePorts},
+			{Name: names.VolumeRestrictions},
+			{Name: names.PodTopologySpread},
+			{Name: names.InterPodAffinity},
+			{Name: names.VolumeBinding},
+			{Name: names.NodeAffinity},
+		},
+	},
+	Filter: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.NodeUnschedulable},
+			{Name: names.NodeName},
+			{Name: names.TaintToleration},
+			{Name: names.NodeAffinity},
+			{Name: names.NodePorts},
+			{Name: names.NodeResourcesFit},
+			{Name: names.VolumeRestrictions},
+			{Name: names.EBSLimits},
+			{Name: names.GCEPDLimits},
+			{Name: names.NodeVolumeLimits},
+			{Name: names.AzureDiskLimits},
+			{Name: names.VolumeBinding},
+			{Name: names.VolumeZone},
+			{Name: names.PodTopologySpread},
+			{Name: names.InterPodAffinity},
+		},
+	},
+	PostFilter: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.DefaultPreemption},
+		},
+	},
+	PreScore: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.InterPodAffinity},
+			{Name: names.PodTopologySpread},
+			{Name: names.TaintToleration},
+			{Name: names.NodeAffinity},
+		},
+	},
+	Score: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.NodeResourcesBalancedAllocation, Weight: 1},
+			{Name: names.ImageLocality, Weight: 1},
+			// Weight is doubled because:
+			// - This is a score coming from user preference.
+			{Name: names.InterPodAffinity, Weight: 2},
+			{Name: names.NodeResourcesFit, Weight: 1},
+			// Weight is doubled because:
+			// - This is a score coming from user preference.
+			{Name: names.NodeAffinity, Weight: 2},
+			// Weight is doubled because:
+			// - This is a score coming from user preference.
+			// - It makes its signal comparable to NodeResourcesLeastAllocated.
+			{Name: names.PodTopologySpread, Weight: 2},
+			// Weight is tripled because:
+			// - This is a score coming from user preference.
+			// - Usage of node tainting to group nodes in the cluster is increasing becoming a use-case
+			//	 for many user workloads
+			{Name: names.TaintToleration, Weight: 3},
+		},
+	},
+	Reserve: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.VolumeBinding},
+		},
+	},
+	PreBind: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.VolumeBinding},
+		},
+	},
+	Bind: config.PluginSet{
+		Enabled: []config.Plugin{
+			{Name: names.DefaultBinder},
+		},
+	},
+}
+
+// PluginConfigsV1beta3 default plugin configurations. This could get versioned, but since
+// all available versions produce the same defaults, we just have one for now.
```

Comment:
If this comment is still true, why are we adding a new configuration?

Perhaps the variable should be named `PluginsConfig` (without a version)

Reply:
I think the 2nd line can be removed. It's better to have defaults tested via version. I agree that the comment is wrong in the context.